sparsity/sparse_frame.py

```diff
@@ -164,10 +164,15 @@ def max(self, *args, **kwargs):
     def min(self, *args, **kwargs):
         return self.data.min(*args, **kwargs)
 
-    def copy(self, *args, **kwargs):
-        return SparseFrame(self.data.copy(*args, **kwargs),
-                           self.index.copy(*args, **kwargs),
-                           self.columns.copy(*args, **kwargs))
+    def copy(self, *args, deep=False, **kwargs):
+        if deep:
```

Comment:
I think deep should be True by default. I use `copy` when I want to be sure that modification to the copy does not change the original df. Now I'd have to remember to add deep=True. Pandas' default is true.

Reply:
Yep you are right!